contribute/style-guides/storybook.md

```diff
@@ -0,0 +1,175 @@
+
+With the CSF format the default export defines some general information about the stories in the file
+
+- `title`: Where the component is going to live in the hierarchy
+- `decorators`: a list which can contain wrappers or provide context, such as theming
+
+```jsx
+// In MyComponent.story.tsx
+
+import MyComponent from './MyComponent';
+
+export default {
+  title: 'General/MyComponent',
+  component: MyComponent,
+  decorators: [ ... ],
+}
+
+```
+
+When it comes to writing the actual stories, you continue in the same file with named exports - which are turned into the story name.
+
+```jsx
+// Will produce a story name “some story”
+export const someStory = () => <MyComponent />;
+```
+
+If you want to write cover cases with different values for props you don’t need to create a new story - using Knobs is usually enough. This will be covered further down.
+
+## Categorization
+
+We currently have these categories:
+
+- Docs Overview - Guidelines and information regarding the design system
+- Forms - Components commonly used in forms such as different kind of inputs
+- General - Components which can be used in a lot of different places
+- Visualizations - Data visualizations
+- Panel - Components belonging to panels and panel editors
+
+# Writing MDX documentation
+
+## Link the MDX file to a component’s stories
+
+An MDX file is basically a markdown file with the possibility to add jsx. These files are used by Storybook to create a “docs” tab. To link a component’s stories with an MDX file you have to do this:
+
+```jsx
+// In TabsBar.story.tsx
+
+import { TabsBar } from "./TabsBar";
+
+// Import the MDX file
+import mdx from "./TabsBar.mdx";
+
+export default {
+  title: "General/Tabs/TabsBar",
+  component: TabsBar,
+  parameters: {
+    docs: {
+      // This is the reference required for the MDX file
+      page: mdx,
+    },
+  },
+};
+```
+
+## The structure of an MDX file
+
+There are some things that the MDX file should contain:
+
+When and why the component should be used
+Best practices - dos and don’ts for the component
+Usage examples with code. It is possible to use the `Preview` tag to show live examples in MDX
+Props table. This can be generated by doing the following
+
+```jsx
+// In MyComponent.mdx
+
+import { Props } from "@storybook/addon-docs/blocks";
+import { MyComponent } from "./MyComponent";
+
+<Props of={MyComponent} />;
+```
+
+### MDX file without a relationship to a component
+
+An MDX file can exist by itself without any connection to a story. This can be good for writing things such as a general guidelines page. Something that is required when the MDX file has no relation to a component is a `Meta` tag that says where in the hierarchy the component will live. It can look like this
```

Comment:
```suggestion
An MDX file can exist by itself without any connection to a story. This can be good for writing things such as a general guidelines page. Something that is required when the MDX file has no relation to a component is a `Meta` tag that says where in the hierarchy the component will live. It can look like this:
```